cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaReconciler.java

```diff
@@ -682,15 +678,7 @@ protected Future<Void> podDisruptionBudget() {
      *
      * @return  Completes when the PDB was successfully created or updated
      */
-    protected Future<Void> podDisruptionBudgetV1Beta1() {
-        if (pfa.hasPodDisruptionBudgetV1()) {
```

Comment:
The `pfa.hasPodDisruptionBudgetV1()` should not be needed anymore. So it should be removed from the `PlatformFeaturesAvailability` class as well. You should also update the `podDisruptionBudget()` method above to not use it either.